hearthstone/simulator/core/card_pool.py

```diff
@@ -2624,6 +2624,27 @@ def handle_event_powers(self, event: CardEvent, context: Union[BuyPhaseContext,
                         context.owner.play_blood_gem(card)
 
 
+class Shudderling(MonsterCard):
+    not_in_pool = True
+    tier = 1
+    monster_type = MONSTER_TYPES.NEUTRAL
+    base_attack = 1
+    base_health = 1
+
+    def base_battlecry(self, targets: List[MonsterCard], context: BuyPhaseContext):
+        battlecry_minion_queue = [card for card in context.owner.minions_played if card.battlecry and type(card) != type(self)]
+        while len(battlecry_minion_queue) > 0:
+            next_minion = battlecry_minion_queue.pop()
```

Comment:
From what I understand this could just be a basic for loop through the list?

Reply:
🙃